.cspell/cspell.json

```diff
@@ -76,6 +76,10 @@
       "filename": "/core/modules/simpletest/tests/file.test",
       "ignoreWords": ["NONEXISTINGFILENAME", "ufail"]
     },
+    {
+      "filename": "/core/modules/simpletest/simpletest.install",
+      "ignoreWords": ["libssh"]
+    },
```

Comment:
I would be fine with adding this to our custom dictionary, as we may use that word elsewhere in the future.

Reply:
...now you're getting CSpell failures again because the path where these words are used changed to `/core/modules/system/system.install`. So yeah, better to have the word added in `.cspell/backdrop.dic` instead.

Reply:
Gotcha. Done.